src/net/messages.rs

```diff
@@ -0,0 +1,124 @@
+use heapless::{consts, String, Vec};
+use serde::Serialize;
+
+use core::fmt::Write;
+
+#[derive(Debug, Copy, Clone)]
+pub enum SettingsResponseCode {
+    NoError = 0,
+    NoTopic = 1,
+    InvalidPrefix = 2,
+    UnknownTopic = 3,
+    UpdateFailure = 4,
+}
```

Comment:
Perhaps we should define these codes externally so that we can have one shared definition between this module and the receiver that interprets these codes? Also it looks like miniconf.py (right now) doesn't care about the value of the code except that it is non-zero, is there a need for these other values?